client/reader/stream/index.jsx

```diff
@@ -365,6 +370,7 @@ class ReaderStream extends React.Component {
 		return (
 			<PostLifecycle
 				key={ itemKey }
+				ref={ itemKey /* The ref is stored into `InfiniteList`'s `this.ref` map */ }
```

Comment:
tangent: reworking infinite list to use modern refs if going to be an interesting challenge.

Reply:
We already did something like that three or four times, and it's not that hard. In cases like this, you want to replicate the `this.refs` map by implementing your own:
```jsx
itemElements = new Map();

storeElementRef( itemKey ) {
  return el => {
    if ( el ) {
      this.itemElements.set( itemKey, el );
    } else {
      this.itemElements.delete( itemKey );
    }
  }
}

render() {
  return (
    <ul>
      {
        this.props.items.map( item => (
          <li ref={ this.storeElementRef( item.id ) }>{ item.text }</li>
        )
      }
    </ul>
  );
}
```